src/config/address.cpp

```diff
@@ -59,7 +59,7 @@ address::address(const messages::address_item::cptr& message) NOEXCEPT
 }
 
 address::address(const asio::endpoint& uri) NOEXCEPT
-  : address(endpoint{ uri })
+  : address(address{ endpoint{ uri } })
```

Comment:
I'm having a hard time figuring out why this should make a difference, as there are just two constructions of the same `address` object, where the outer is a move construction and the inner is a `network::address` construction from a `network::endpoint` object (which is what it was before the change). Am I missing something?